onnx_tf/handlers/frontend/square.py

```diff
@@ -0,0 +1,35 @@
+import copy
+import numpy as np
+from onnx import helper
+from onnx import TensorProto
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import tf_op
+from onnx_tf.handlers.frontend.multiply import Multiply
+
+
+@tf_op("Square")
+class Square(FrontendHandler):
+
+  @classmethod
+  def version_1(cls, node, **kwargs):
+    attr = copy.deepcopy(node.attr)
+    out_shapes = np.array(attr.pop('_output_shapes', None))
+    out_shapes_list = list()
+    i = 0
+    for o in out_shapes:
+      out_shapes_list.append(
+          helper.make_tensor(
+              name='output_{}'.format(i),
+              data_type=TensorProto.INT32,
+              dims=o.shape,
+              vals=o))
+      i += 1
+    attr['_output_shapes'] = out_shapes_list
```

Comment:
I don't think you need to compute output shape.

Reply:
@tjingrant the output shape is added here to remove the following warning message:
onnx-tensorflow/onnx_tf/handlers/frontend_handler.py:141: UserWarning: _output_shapes is not in node.attr. The num of output is set to 1 for commonly. It will cause problem with case of multiple outputs. 

I know in my case here, there should only have one output, so the default behavior is OK, but I don't want to get this warning message every time I call Square

Reply:
@winnietsang Because you pop `_output_shapes` from attr at L16.

Reply:
@fumihwh If I just send in the attr directly like the following code:
    mul_node = Multiply.handle(
        helper.make_node(
            'Mul',
            inputs=[node.inputs[0], node.inputs[0]],
            outputs=cls.get_outputs_names(node),
            name=node.name,
            **(node.attr)), **kwargs)

will get this error message:
  File "/root/programs/onnx/onnx/helper.py", line 51, in <genexpr>
    for key, value in sorted(kwargs.items()))
  File "/root/programs/onnx/onnx/helper.py", line 246, in make_attribute
    "You passed in an iterable attribute but I cannot figure out "
ValueError: You passed in an iterable attribute but I cannot figure out its applicable type.

That's why I need to pop _output_shapes first and put the _output_shapes value into a tensor list then put it back into attr. With this change then helper.make_attribute can process the value for _output_shapes.

Reply:
@fumihwh I always feel like this method of passing a onnx protobuf where the handler is expecting a tf protobuf is problematic. We should never use onnx.helper.make_node here.

I guess we have to deal with it now.

Reply:
@winnietsang As an adhoc fix, `**(node.attr)` does not need to pass to `helper.make_node`.
@tjingrant `TensorflowNode` not good enough?

Reply:
@tjingrant and @fumihwh  I updated the square.py to create a TensorflowNode directly and then send this TensorflowNode to the Multiply handler. Does this satisfy both of your concern?

Reply:
@winnietsang thanks, this makes a lot more sense to me now.

Reply:
I also think using TensorflowNode is cleaner than using ONNX node. Just wondering if we should go back to refactor some of the existing handlers.

Reply:
Hi @chinhuang007 i think we should go back and refactor the usage of existing handlers.